src/components/ReadMeDialog/ReadMePdf.js

```
@@ -0,0 +1,124 @@
+  };
+
+  const renderContent = (text) => {
+    // split text with newline and generate pdf elements
+    const lines = text.split(/\r?\n/);
+    const pdfContent = lines.map((item) => {
+      const count = (item.match(/#/g) || []).length;
+      if (count > 0) {
+        return renderSubHeader(item.replaceAll('#', ''), count);
+      }
+      return (
+        <Text style={styles.content}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)